xblock/test/test_fields.py

```diff
@@ -780,7 +810,11 @@ def test_to_string_regexp_matches(self, _type, value, regexp):
                   kaw: null
             """),
             [1, 2.345, {"foo": True, "bar": [1, 2, 3]}, {"meow": False, "woof": True, "kaw": None}]
-        )
+        ),
+        # Test that legacy DateTime format including double quotes can still be imported for compatibility with
+        # old data export tar balls.
+        (DateTime, '"2014-04-01T02:03:04.567890"', dt.datetime(2014, 4, 1, 2, 3, 4, 567890).replace(tzinfo=pytz.utc)),
+        (DateTime, '"2014-04-01T02:03:04.000000"', dt.datetime(2014, 4, 1, 2, 3, 4).replace(tzinfo=pytz.utc)),
     )
```

Comment:
Since a datetime string *without* double quotes should also pass this test, perhaps add a couple of those to underscore that fact? I know that it's tested in `test_both_directions` above. I'm not even sure why `test_from_string` can't just be combined into `test_both_directions`. This new test case is perhaps the only one that can't be combined into `test_both_directions` as it won't be symmetric.